eventual-cassandra/src/main/scala/com/evolutiongaming/kafka/journal/eventual/cassandra/JournalHead.scala

```diff
@@ -6,7 +6,13 @@ import com.evolutiongaming.kafka.journal.{DeleteTo, PartitionOffset, SeqNr}
 import com.evolutiongaming.scassandra.syntax._
 import com.evolutiongaming.scassandra.{DecodeRow, EncodeRow}
 
-
+/** Represent metadata of the latest journal event saved to Cassandra.
+  *
+  * @param partitionOffset the offset where the event is located in Kafka
```

Comment:
it's not about event only, but any relevant record to this journal, and there are records like:

```
Append(events)
Delete
Purge
Mark
```